components/pedigree/resources/src/main/resources/pedigree/model/positionedGraph.js

```diff
@@ -13,6 +14,7 @@ define([
         "pedigree/view/ageCalc"
     ], function(
         PedigreeDate,
+        DynamicPositionedGraph,
```

Comment:
`DynamicPositionedGraph` is only used for (old) testing/.debug code, the correct change is to move the code that uses it